xl-v1-utils.go

```diff
@@ -24,6 +24,33 @@ import (
 	"time"
 )
 
+// Validates if we have quorum based on the errors.
+func isQuorum(errs []error, minQuorumCount int) bool {
+	var diskNotFoundCount int
+	for _, err := range errs {
+		if err == errDiskNotFound {
+			diskNotFoundCount++
+		}
+	}
+	if diskNotFoundCount > len(errs)-minQuorumCount {
+		return false
+	}
+	return true
+}
+
```

Comment:
logic based on diskFoundCount instead of diskNotFoundCount is more readable, like:
```
func isQuorum(errs []error, minQuorumCount int) bool {
        var diskFoundCount int
        for _, err := range errs {
                if err != errDiskNotFound {
                        diskFoundCount++
                }
        }
        return diskFoundCount >= minQuorumCount
}
```